easybuild/easyconfigs/p/Python/Python-2.7.10-GCC-4.9.3-2.25-bare.eb

```diff
@@ -0,0 +1,31 @@
+name = 'Python'
+version = '2.7.10'
+versionsuffix = '-bare'
+
+homepage = 'http://python.org/'
+description = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
+
+toolchain = {'name': 'GCC', 'version': '4.9.3-2.25'}
```

Comment:
I like the rule of thumb you mentioned with treating GCCcore like we treat using the dummy/system compiler.

I also like your idea w.r.t. Python: maybe open a separate issue on that, to have a handle for this?